openslides_backend/shared/schema.py

```diff
@@ -26,7 +26,7 @@
 }
 optional_str_schema: Schema = {
     "type": ["string", "null"],
-    "minLength": 1,
+    "minLength": 0,
```

Comment:
Why is it necessary to change one of the basic schemas?
The other optional schemas still have a `minLength` of 1. This is an inconsistency, that will cause confusion during future development. Either change them all (impractical, since that'll change the functionality of quite a few actions), or change none of them and write this code directly in the class that needs it.

Reply:
That's just a remnant of me experimenting on something. Thank you for pointing that out. :) I will revert that.